src/DaPigGuy/PiggyCustomEnchants/enchants/armor/HeavyEnchant.php

```diff
@@ -22,14 +22,6 @@ class HeavyEnchant extends ReactiveEnchantment
     /** @var string */
     public $name = "Heavy";
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 20, so I commented this line